docs/ibis-for-dplyr-users.ipynb

```diff
@@ -0,0 +1,777 @@
+{
```

Comment:
<p>Most databases allow you to choose null ordering either as a global configuration option, in the expression itself (e.g., <code>NULLS LAST</code>) or both. So far nobody has asked for this functionality so we haven't implemented it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6050/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Oh neat. This honestly isn't something I've ever had to deal with in database-land. <code>NULLS LAST</code> is good to know about. </p>